exporters/stdout/stdoutlog/exporter_test.go

```diff
@@ -0,0 +1,228 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package stdoutlog_test // import "go.opentelemetry.io/otel/exporters/stdout/stdoutout"
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutlog"
+	"go.opentelemetry.io/otel/log"
+	sdklog "go.opentelemetry.io/otel/sdk/log"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func TestExporter(t *testing.T) {
+	var buf bytes.Buffer
+
+	exporter, err := stdoutlog.New(stdoutlog.WithWriter(&buf))
+	assert.NoError(t, err)
+	assert.NotNil(t, exporter)
+
+	now := time.Now()
+	record := getRecord(now)
+
+	// Export a record
+	err = exporter.Export(context.Background(), []sdklog.Record{record})
+	assert.NoError(t, err)
+
+	// Check the writer
+	assert.Equal(t, expectedJSON(now, false), buf.String())
+
+	// Flush the exporter
+	err = exporter.ForceFlush(context.Background())
+	assert.NoError(t, err)
+
+	// Shutdown the exporter
+	err = exporter.Shutdown(context.Background())
+	assert.NoError(t, err)
+
+	// Export a record after shutdown, this should not be written
+	err = exporter.Export(context.Background(), []sdklog.Record{record})
+	assert.NoError(t, err)
+
+	// Check the writer
+	assert.Equal(t, expectedJSON(now, false), buf.String())
+}
+
+func TestExporterExport(t *testing.T) {
+	now := time.Now()
+
+	record := getRecord(now)
+	records := []sdklog.Record{record, record}
+
+	// Get canceled context
+	canceledCtx, cancel := context.WithCancel(context.Background())
+	cancel()
+
+	testCases := []struct {
+		name           string
+		options        []stdoutlog.Option
+		ctx            context.Context
+		records        []sdklog.Record
+		expectedResult string
+		expectedError  error
```

Comment:
Can we use `want` term instead of `expected`?

References:
- https://go.dev/wiki/CodeReviewComments#useful-test-failures
- https://google.github.io/styleguide/go/decisions#got-before-want